index.html

```diff
@@ -7,33 +7,241 @@
+          </ul>
+          <div class="pound css-background-img"></div>
+        </ul>
+        <ul class="box-container bg-redish">
+          <li class="title-sub white">Frameworks</li>
+          <ul class="list-frameworks">
+            <li class="white">React.js</li>
+            <li class="white">Ruby on rails</li>
+            <li class="white">RSpec</li>
+            <li class="white">Capibara</li>
+          </ul>
+          <div class="card css-background-img"></div>
+        </ul>
+        <ul class="box-container bg-yellowish">
+          <li class="title-sub white">Skills</li>
+          <ul class="list-skills">
+            <li class="white">Database Management</li>
+            <li class="white">CLI</li>
+            <li class="white">API Design</li>
+            <li class="white">Version Control</li>
+            <li class="white">Web Development</li>
+          </ul>
+          <div class="idea css-background-img"></div>
+        </ul>
+      </div>
+    </div>
+  </section>
+
+  <!--Get Started Section-->
+  <div class="bg-offwhite">
+    <div class="top-edge-container">
+      <div class="geo-background-img"></div>
+    </div>
+    <section class="section-container">
+      <div class="section-sub">
+        <h1 class="title dark-blue">Get started</h1>
+        <div class="contents">
+          <p class="paragraph dark-blue">
+            I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.
+          </p>
+        </div>
+        <p class="small-title offwhite">Your name</p>
+        <div>
+          <div class="input-button dark-blue bg-white">Anastasiia</div>
+        </div>
+        <p class="small-title offwhite">Your email</p>
+        <div class="input-button bg-white dark-blue">anastasiia@gm...</div>
+        <p class="small-title offwhite">Your message</p>
+        <div class="bigger-box bg-white"></div>
+        <button class="collab-button bg-dark-blue white">Start collaboration</button>
+      </div>
+      <div class="section-bottom bg-dark-blue">
+        <div class="getmy">
+          <p class="project-goes greenish">Get my resume</p>
+          <img class="point-up" src="images/point-up.png" alt="This is an arrow pointing to the upper right corner." width="22" height="22">
+        </div>
+        <div class="icons">
+          <img src="images/stackoverflow.png" alt="This is a stackoverflow icon." width="21" height="16">
+          <img src="images/github.png" alt="This is the Github icon." width="24" height="24">
+          <img src="images/twitter.png" alt="This is the Twitter icon." width="24" height="24">
+          <img src="images/linkedin.png" alt="This is LinkedIn icon." width="24" height="24">
+          <img src="images/m.png" alt="This is an M icon." width="24" height="24">
+          <img src="images/peace.png" alt="This is a V-sign icon." width="24" height="24">
+        </div>
+      </div>
+    </section>
+  </div>
```

Comment:
Great job with your footer! I have some suggestions to make it even better and more consistent with the [Figma template](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?type=design&node-id=23-9&t=DOmVpbk2buQv96CX-0):

- [ ] Please align the icon on your `Get my resume` button to make it look more centered. 👍
- [ ] Check the height and width **of all of your social icons in your `footer` to ensure they have consistent dimensions**.  💯

  _By applying these suggestions you will make your footer look more organized and visually pleasing._ 🎯

  ### Your design:
  ![image](https://user-images.githubusercontent.com/107952724/236647907-8faa25c9-1feb-489f-bd13-255e45d531d6.png)
  ### Figma design:
  ![image](https://user-images.githubusercontent.com/107952724/236647928-7e665e37-bb6f-4501-86b3-1d534b6f8e44.png)